Update neofs-local with neofs-node 0.2.x version

alexvanin opened the pull request:
This pull requests updates all docker images for 0.2.x version of neofs-node and neofs-cli. There is an updated how-to with relevant examples. 

Configure file is also changed to fix #15

alexvanin committed:
Update neofs-node and neofs-cli to 0.2.x version

With 0.2.x release NeoFS supports instruments for developers
like `neofs-proto` library to build native applications for NeoFS.
Therefore we separated CLI, http gateway and drop service from
NeoFS repository as an independent entities.

alexvanin committed:
Update how-to file and readme files

volekerb commented on .env:
```diff
@@ -14,7 +14,8 @@ GRAFANA_VERSION=6.3.5
 # NeoFS images versions
-NEOFS_IMAGES_VERSION=0.1.11
+NEOFS_IMAGES_VERSION=0.2.3
+NEOFS_CLI_VERSION=0.2.1
```
not that obvious from the commit why CLI and IMAGES now have different versions.

alexvanin replied:
In the development we are trying to separate neofs-node repo into group of small libraries and applications. 0.2.x releases support `neofs-proto` library, that allows to build applications for NeoFS, e.g. NeoFS CLI. 

Do you think we need explicit mentions about this?

volekerb replied:
at least in commit message it will be nice to have, thank you! =)

alexvanin replied:
Alright! I will add more info about this in commit message and force-push the changes into this branch.

volekerb commented on neofs-local/HOWTO.md:
```diff
@@ -41,84 +45,77 @@ ERROR: error pulling image configuration: Get https://registry-1.docker.io/v2/li
 NEOFS_BOOKKEEPER_CONTAINER_NODE_COUNT_FACTOR=0
 NEOFS_BOOKKEEPER_CONTAINER_CONTAINER_CAPACITY_FACTOR=0
-
-# Do not delete unpaid objects
```
that comment not relevant anymore, right?

alexvanin replied:
Yes. Actually `.env` file never had this comment.

volekerb commented on neofs-local/HOWTO.md:
```diff
@@ -169,168 +169,168 @@ drwxr-xr-x    1 root     root        4.0K Sep  2 10:04 ..
 bash-5.0# env
 NEOFS_CLI_KEY=/user.key
-NEOFS_CLI_ADDRESS storage01:8080
+NEOFS_CLI_ADDRESS=storage10:8080
```
hm. why is that now storage10?

alexvanin replied:
CLI app is dependent on `storage10` node, look at `neofs-local/docker-compose.cli.yml` :
```yml
    environment:
      - NEOFS_CLI_KEY=/user.key
      - NEOFS_CLI_ADDRESS=storage10:8080
```

I think it will be better to work with this node in all examples. However there is no difference which node to use for queries. What do you think?

volekerb replied:
oh I see now. True, agreed.

volekerb commented on neofs-local/HOWTO.md:
````diff
@@ -408,48 +408,27 @@ $
 
 ```
-$ make local_down
+$ make local_down 
````
nit: space in the end of the line probably could be removed

im-kulikov commented on neofs-local/docker-compose.yml:
```diff
@@ -6,15 +6,16 @@ networks:
       storage01:
         condition: service_healthy
     environment:
       - GW_REQUEST_TIMEOUT=40h
```
it's too long, I think, that 5-10m will be enough, what you think?

alexvanin replied:
Yeah, seems so. I will change timeout duration in next commit.

alexvanin committed:
Reduce timeout duration on http gateway backend

Previously there was high timeout duration for debug
purposes. Now it is reasonable to set it much lower in sense
of real-world usage.
